index.html

```diff
@@ -4,484 +4,528 @@
+          </ul>
+        </nav>
+        <nav class="menu nav-menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#home">Home</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#categories">Categories</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#service">Service</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#jobs">Jobs</a>
+            </li>
+          </ul>
+        </nav>
 
-                <!-- Search button -->
-                <div class="hero-section__button">
-                  <a class="buttons buttons__search-button" href="#">
-                    Search
-                  </a>
-                </div>
-              </div>
-            </div>
+        <!-- Navigation button -->
+        <button class="button button__cta" href="#">Post a vacancy</button>
+      </div>
+    </header>
 
-            <!-- Right wrapper -->
-            <div class="right-wrapper">
-              <img
-                class="right-wrapper__img"
-                src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
-                alt="satisfied-lovely-woman-holds-modern-cell-phone"
-              />
+    <main>
+      <!-- Hero section -->
+      <section class="hero">
+        <!-- Hero section wrapper -->
 
-              <!-- Big ellipse -->
-              <img
-                class="right-wrapper__ellipse-big"
-                src="./src/img/ellipse-big.svg"
-                alt="big ellipse"
-              />
-              <!-- Small ellipse -->
-              <img
-                class="right-wrapper__ellipse-small"
-                src="./src/img/ellipse-small.svg"
-                alt="small ellipse"
-              />
+        <!-- Info -->
+        <div class="info">
+          <h1 class="info__title title">Find & Apply For Your Dream Job</h1>
+          <p class="info__subtitle">
+            The website is chockablock with useful features, allowing you to
+            efficiently search and apply for roles across all sectors,
+            experience levels and locations.
+          </p>
+
+          <!-- Search container -->
+          <div class="search-container">
+            <!-- Search form -->
 
-              <!-- Flying icons -->
-              <div class="right-wrapper__flying-icon google-icon">
+            <form class="search-container__search-form" method="get">
+              <div class="search-area">
                 <img
-                  class="google-img"
-                  src="./src/icons/flat-color-icons_google.svg"
-                  alt="google icon"
+                  class="search-container__search-icon"
```

Comment:
```suggestion
                  class="search-icon"
```
And the icon may be used anywhere